src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestDatabase.scala

```diff
@@ -0,0 +1,53 @@
+package uk.gov.nationalarchives.tdr.api.utils
+
+import java.sql.Connection
+
+import org.scalatest.{BeforeAndAfterEach, Suite}
+import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import uk.gov.nationalarchives.tdr.api.service.FileMetadataService.clientSideProperties
+import uk.gov.nationalarchives.tdr.api.service.TransferAgreementService.transferAgreementProperties
+import uk.gov.nationalarchives.tdr.api.utils.TestUtils.{addConsignmentProperty, addFileProperty}
+
+/**
+ * This trait should be mixed into specs which access the test database.
+ *
+ * It provides a test database connection and cleans up all of the test data after every test.
+ */
+trait TestDatabase extends BeforeAndAfterEach {
+  this: Suite =>
+
+  val databaseConnection: Connection = DbConnection.db.source.createConnection()
+
+  override def beforeEach(): Unit = {
+    databaseConnection.prepareStatement("DELETE FROM FileMetadata").execute()
+    databaseConnection.prepareStatement("DELETE FROM FileProperty").execute()
+    databaseConnection.prepareStatement("DELETE FROM FFIDMetadataMatches").execute()
+    databaseConnection.prepareStatement("DELETE FROM FFIDMetadata").execute()
+    databaseConnection.prepareStatement("DELETE FROM AVMetadata").execute()
+    databaseConnection.prepareStatement("DELETE FROM File").execute()
+    databaseConnection.prepareStatement("DELETE FROM ConsignmentMetadata").execute()
+    databaseConnection.prepareStatement("DELETE FROM ConsignmentProperty").execute()
+    databaseConnection.prepareStatement("DELETE FROM Consignment").execute()
+    databaseConnection.prepareStatement("DELETE FROM Series").execute()
+    databaseConnection.prepareStatement("DELETE FROM Body").execute()
```

Comment:
I wanted to do something nicer here by using `DbConnection.db.run(DBIO.seq())` to run multiple queries, but it returns a Future, which is quite hard to get working with `beforeEach`. Might be worth revisiting at some point.